mtp_api/apps/mtp_auth/tests/test_views.py

```diff
@@ -0,0 +1,51 @@
+from django.core.urlresolvers import reverse
+
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from core.tests.utils import make_test_users, make_test_oauth_applications
+
+from prison.models import Prison
+
+
+class UserViewTestCase(APITestCase):
+    fixtures = ['test_prisons.json']
+
+    def setUp(self):
+        super(UserViewTestCase, self).setUp()
+        self.users = make_test_users(users_per_prison=2)
+        self.prisons = Prison.objects.all()
+        make_test_oauth_applications()
+
+    def _get_url(self, username):
+        return reverse('user-detail', kwargs={'username': username})
+
+    def test_401_if_not_logged_in(self):
```

Comment:
Might be more descriptive to call this `test_cannot_access_data_when_not_logged_in`?